docker-vertica-v2/Dockerfile

```diff
@@ -161,7 +162,8 @@ RUN set -x \
   && tar -C / -Jxpf /tmp/s6-overlay-noarch.tar.xz \
   # Create a symlink to the rsync for use with vbr. This works around a problem
   # seen in some deployments where vbr cannot find rsync.
-  && ln -s /opt/vertica/bin/rsync /usr/bin/rsync
+  && ln -s /opt/vertica/bin/rsync /usr/bin/rsync \
+  && sh /tmp/cleanup.sh
 
 ENTRYPOINT [ "/init" ]
 
```

Comment:
Can you update the LABEL a few lines below? The one `org.opencontainers.image.source` should point to the correct path. Also, we should add a label to indicate the deployment method (we would need one in the admintools image as well). Perhaps `vertica-deployment-method`?